p2p/vpacket.go

```diff
@@ -1,17 +1,19 @@
 /**
-*  @file
+*  Package p2p
 *  @copyright defined in go-seele/LICENSE
  */
 package p2p
 
-import "encoding/binary"
+import  (
+    "encoding/binary"
+)
 
 const (
     //VPacketHeadLen head length in net-package
```

Comment:
Add a space at the beginning, pay attention to others

Reply:
nice suggestion. fixed it.